include/SFML/Audio/SoundSource.hpp

```diff
@@ -33,6 +33,7 @@
 #include <SFML/System/Vector3.hpp>
 
 
+// NOLINTBEGIN(readability-make-member-function-const)
```

Comment:
This could be moved inside the namespace like it is done in the corresponding source file.